FreeType.xs

```diff
@@ -628,7 +628,7 @@ qefft2_face_fixed_sizes (Font_FreeType_Face face)
         int i;
         FT_Bitmap_Size *size;
         HV *hash;
-        double pt, ppem;
+        double pt = 12.0, ppem;
```

Comment:
I'll not merge that. Because:
1. if it crashed, please, try to provide some test case that proves it. - I'll merge it then.
2. Otherwise, you introduce new default "pt size = 12".  It will might break existing code.